phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -2514,7 +2529,7 @@ private void createSysMutexTable(HBaseAdmin admin) throws IOException, SQLExcept
         return Lists.newArrayList(admin.listTableNames(QueryConstants.SYSTEM_SCHEMA_NAME + "\\..*"));
     }
 
-    private void createOtherSystemTables(PhoenixConnection metaConnection) throws SQLException {
+    private void createOtherSystemTables(PhoenixConnection metaConnection, HBaseAdmin hBaseAdmin) throws SQLException {
```

Comment:
nit: s/hBaseAdmin/hbaseAdmin/ camel-case typically isn't observed on "hbase" ;)

Reply:
Will do.

Reply:
Added it in a new commit.